src/cmd/promremotebench/checker.go

```diff
@@ -104,6 +119,44 @@ func (c *checker) GetHostNames() []string {
 	return results
 }
 
+func (c *checker) cleanupLoop(loopDuration time.Duration) {
+	hostsToRemove := make([]string, 0, 4)
+	hostsToTrim := make([]string, 0, 20)
+
+	for {
+		time.Sleep(loopDuration)
```

Comment:
Maybe better to use a [ticker here](https://gobyexample.com/tickers)?

Reply:
Sure